crates/core/src/v1/connection.rs

```diff
@@ -89,7 +89,7 @@ impl Connection {
     /// Convenience method to run a prepared statement query.
     /// ## Example
     ///
-    /// ```rust,no_run
+    /// ```rust,no_run,ignore
```

Comment:
Ignored for now since these are no longer public, we will eventually completely remove some of these APIs as we will only require what is needed for the async api.